README.md

```diff
@@ -6,6 +6,8 @@ SqlPackage.exe is a command line interface to DacFx and is available for Windows
 
 Microsoft.Build.Sql (preview) is a [.NET project SDK](https://docs.microsoft.com/dotnet/core/project-sdk/overview) for SQL projects, compiling T-SQL code to a data-tier application package (dacpac).  The SQL project SDK is available via NuGet ([https://www.nuget.org/packages/Microsoft.Build.Sql/](https://www.nuget.org/packages/Microsoft.Build.Sql/)) and in this [repository](/src/Microsoft.Build.Sql/README.md).
 
+Microsoft.Build.Sql.Templates (preview) is a set of [.NET project templates](https://learn.microsoft.com/dotnet/core/tools/custom-templates) for SQL projects.  The SQL project SDK templates are available via NuGet ([https://www.nuget.org/packages/Microsoft.Build.Sql.Templates/](https://www.nuget.org/packages/Microsoft.Build.Sql.Templates/)).
+
 ## Repository Focus
```

Comment:
This title is not needed. The previous paragraph explains this repository (that this readme file is in) is related to Microsoft.Build.Sql.Templates. This title confuses that fact. If the thought is to clarify the scope and purpose of the Microsoft.Build.Sql.Templates project it would be better to add more content to the definition above.